net-im/mattermost-desktop-bin/mattermost-desktop-bin-5.2.0-r1.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="${PN%-*}"
+MY_PV="${PV/_rc/-rc}"
+
+PYTHON_COMPAT=( python3_{8..11} )
+inherit desktop python-single-r1 xdg
+
+DESCRIPTION="Mattermost Desktop application"
+HOMEPAGE="https://mattermost.com/"
+
+SRC_URI="https://releases.mattermost.com/desktop/${MY_PV}/mattermost-desktop-${MY_PV}-linux-x64.tar.gz"
+
+LICENSE="Apache-2.0 GPL-2+ LGPL-2.1+ MIT"
+SLOT="0"
+# Starting with 5.2.0 upstream dropped x86 for their binary release #879519
+KEYWORDS="~amd64"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
```

Comment:
I don't think this is enough, we don't want it to point to "python3" (see bug). It'll fail with -native-symlinks bur it also means PYTHON_SINGLE_TARGET choice by the user isn't respected.

Reply:
So should I also update the symlink?

Reply:
Yes